definitions/ext-nas/readynas-dashboard.json

```diff
@@ -53,7 +53,7 @@
 						"nrqlQueries": [
 							{
 								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.temperatureValue) AS 'Celsius', (latest(kentik.snmp.temperatureValue)*1.8) + 32 AS 'Fahrenheit' WHERE provider = 'kentik-nas' AND SysObjectID = '.1.3.6.1.4.1.4526.100.16'"
+								"query": "FROM Metric SELECT latest(kentik.snmp.temperatureValue) AS 'Celsius' (latest(kentik.snmp.temperatureValue)*1.8) + 32 AS 'Fahrenheit' WHERE provider = 'kentik-nas' AND SysObjectID = '.1.3.6.1.4.1.4526.100.16'"
```

Comment:
can you revert this one back? this is a known issue in the script that we will fix next week